### PR TITLE
 dom-event-delegation -- Sarah You

### PR DESCRIPTION
**Pull request by sarah-you**
https://user-images.githubusercontent.com/103456134/230495016-81f0b04c-7462-4b37-a7d7-f6b30fe92c4a.mov

### CODE REVIEW

**Comment by sarah-you** [comment]
dom-event-delegation

File: dom-event-delegation/dom-event-delegation-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# dom-event-delegation-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `event.target`?
+the element you are adding the event listener to;
```
Comment (shawnkost) [inline dom-event-delegation/dom-event-delegation-quiz-notes.md:8]: Not quite! `event.target` is the element that was interacted with to trigger the event. `event.currentTarget` would be the element that has the event listener attached to it.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested on your quiz answer!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**